shared/modules/persistence_module.go

```diff
@@ -33,6 +38,22 @@ type PersistenceModule interface {
 	HandleDebugMessage(*messaging.DebugMessage) error
 }
 
+// TreeStore is fulfilled by the treeStore to create an
```

Comment:
`by the treeStore` - What is the `treeStore`?

Reply:
Updated to be more accurate.